### PR TITLE
WLCZ open new combined BB + SM site

### PR DESCRIPTION
**Pull request by Thomas-Benson**
Installing combined BB + SM borehole sensor at new site WLCZ
BB and SM sensors handled seperately:
 - Nanometrics Cascadia Trillium TC120PH-1 (3515)
 - Nanometrics Cascadia Titan TACCL-PH1 (3515)
Using Exisiting Titan and Trillium reponses for new sensor package:
 - `sensor_Nanometrics-Inc_Trillium-Compact-120PH-2`
 - `sensor_Nanometrics-Inc_Titan`

Datalogger is a Centaur (10327), firmware verison 4.8.5

Station coordinates: `-41.304416, 174.782848, 47m`
Sensors are on location codes 10 and 21, at a depth of 15.5m

New streams WLCZ.10.HH-12Z + WLCZ.21.HN-12Z

Surface SM recorder to be installed at a later date

### CODE REVIEW

**Comment by Thomas-Benson** [comment]
Waiting on Delta approval for new sensor naming/treatment...

**Review by CBurton90 — DISMISSED** [review]
Looking good, glad it passed the checks using a single serial number!

**Review by ozym — APPROVED** [review]
nice

File: assets/sensors.csv
Comment (mnaguit) [inline assets/sensors.csv:694]: serial number in the ticket is `3512`?

File: install/sensors.csv
Comment (mnaguit) [inline install/sensors.csv:820]: please double check the serial number, thanks.

File: install/connections.csv
Comment (mnaguit) [inline install/connections.csv:1061]: is there a need to fill up the role for broadband and for strong motion?

File: network/stations.csv
Comment (mnaguit) [inline network/stations.csv:2129]: elevation in the ticket is `38m` instead of `47`?

File: network/sites.csv
Comment (mnaguit) [inline network/sites.csv:2267]: please double check the elevation, thanks @Thomas-Benson 🙏

**Review by mnaguit — COMMENTED** [review]
Great work @Thomas-Benson thanks! Just a request to double check the elevation and the serial number. Otherwise all good, noting the update for azimuth once available.

File: install/sensors.csv
Comment (gva6500) [reply install/sensors.csv:820]: @mnaguit  the serial number(s) is actually correct. The strong motion and weak motion is housed together in a single casing. 

@Thomas-Benson  I made a mistake in the Integration testing ticket.  The down-hole Titan is called TACCl-BH1 so the this should probably called **Cascadia Titan TACCL-BH1**  ( I did fix in the ticket also....)

File: network/stations.csv
Comment (gva6500) [reply network/stations.csv:2129]: The elevation I got from the GPS on the Centaur said 38 at the time. Sits now at 42 (after aqcuiring more satelites) 
Google earth elevation is 47... The truth is proable someshwere in the the middle 🙂

File: install/sensors.csv
Comment (mnaguit) [reply install/sensors.csv:820]: Thanks @gva6500 , had to double check coz the ticket says the serial number is `3512` instead of `3515`. just a typo in the ticket?

File: install/connections.csv
Comment (Thomas-Benson) [reply install/connections.csv:1061]: No need for roles since both sensors are connected to the same datalogger. Roles are only used to distinguish between different dataloggers at the same place

File: install/sensors.csv
Comment (gva6500) [reply install/sensors.csv:820]: You are 100% correct! @mnaguit  
 my short stubby fingers make lots of typos 🙃

File: network/stations.csv
Comment (Thomas-Benson) [reply network/stations.csv:2130]: We'll go with an elevation of 42m then?

**Comment by Thomas-Benson** [comment]
The cascadia naming has been approved by delta... (Sorry for the delay, I was on leave.)

**Review by salichon — APPROVED** [review]
Hi  
@Thomas-Benson  - this looks good to me ! 
a curvy  3 headed ones ... 
well done :0

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**
